source/partial-deep.d.ts

```diff
@@ -10,6 +10,13 @@ export type PartialDeepOptions = {
 	@default false
 	*/
 	readonly recurseIntoArrays?: boolean;
+
+	/**
+	Whether to authorize undefined values in arrays.
```

Comment:
"authorize" is not the correct word here. Also needs more extensive explanation of how it works.

Reply:
Done, I hope the new explanation will be OK, do not hesitate to modify if you think you could provide a better one.